src/components/ec-modal/ec-modal.vue

```diff
@@ -123,6 +131,9 @@ export default {
     isLoadingNegativeButton() {
       return !!this.isLoading.negative;
     },
+    zIndexStyle() {
+      return this.zIndex ? `z-index: ${this.zIndex}` : '';
```

Comment:
We should pass `null` if zIndex is not defined, so Vue will not render the style attribute at all. Returning empty string will render empty style attribute.